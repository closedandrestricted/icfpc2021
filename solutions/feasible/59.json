{"vertices":[[41,80],[48,101],[45,103],[25,91],[41,79],[23,93],[22,66],[53,105],[44,56],[24,82],[11,72],[40,74],[50,77],[49,77],[17,90],[48,96],[59,103],[31,103]]}