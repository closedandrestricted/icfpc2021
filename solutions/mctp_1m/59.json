{"vertices":[[48,66],[70,64],[29,80],[57,43],[53,79],[43,100],[39,61],[71,15],[30,82],[72,127],[58,118],[18,96],[64,42],[51,34],[40,113],[74,99],[42,60],[56,102]]}