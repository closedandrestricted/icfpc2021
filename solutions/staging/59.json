{"vertices":[[81,50],[74,71],[72,72],[49,71],[71,48],[51,60],[48,46],[18,66],[82,68],[42,92],[56,83],[67,79],[45,75],[45,75],[48,100],[14,86],[60,99],[32,89]]}