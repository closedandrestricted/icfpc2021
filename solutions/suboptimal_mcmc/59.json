{"vertices":[[18,67],[36,80],[37,81],[56,95],[14,74],[19,97],[36,80],[80,116],[36,79],[34,60],[30,76],[19,72],[56,101],[56,101],[47,68],[49,36],[65,128],[58,52]]}