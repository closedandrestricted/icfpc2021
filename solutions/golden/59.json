{"vertices":[[57,61],[66,41],[45,81],[81,21],[62,64],[69,86],[65,41],[51,31],[72,43],[75,90],[84,104],[68,61],[40,57],[40,57],[73,89],[59,113],[14,68],[77,108]]}